### DISCUSSION TITLE
为什么我的CustomTable组件照抄Table columns的类型作为props的类型会错误，直接用Table组件却不会发生类型错误

### DISCUSSION THREAD

**Discussion by 704998200** [ant-design/ant-design#47135] [Q&A] [answered]
![image](https://github.com/ant-design/ant-design/assets/49085215/4769f259-41de-4035-9aff-cef3e9a503f0)
![image](https://github.com/ant-design/ant-design/assets/49085215/831885d6-6a14-40cd-954f-72ad10d532a2)
![image](https://github.com/ant-design/ant-design/assets/49085215/f282b617-b116-48d0-9245-3eda32f7f60d)

**Comment by zombieJ** [comment, marked as answer]
没看到错误内容，不好判断。Table 本身是泛型，另外你做定义的时候可以直接从 TableProps 做拓展估计也能好。

**Reply by 704998200** [reply]
谢谢，这么做确实能解决我的问题。钻牛角尖了，忘了可以extends TableProps了。这是我的的错误内容
![image](https://github.com/ant-design/ant-design/assets/49085215/7a920119-9c8c-4acf-a155-339b46d3769e)
